extensions/azurecore/src/account-provider/auths/azureDeviceCode.ts

```diff
@@ -43,7 +46,6 @@ interface DeviceCodeLoginResult {
 }
 
 export class AzureDeviceCode extends AzureAuth {
-
 	private static readonly USER_FRIENDLY_NAME: string = localize('azure.azureDeviceCodeAuth', "Azure Device Code");
```

Comment:
I noticed you changed from " to ' earlier, should the same be done here?

Reply:
Not for localize calls